### PR TITLE
GBNG-4200 - Custom Grade Scales not Sorted Properly

### PR DESCRIPTION
**Pull request by mpellicer**
Order the custom grade scales by value instead of key.

### CODE REVIEW

File: gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/SettingsGradingSchemaPanel.java [hunk 1]
```diff
@@ -300,3 +304,23 @@ public int compare(final String o1, final String o2) {
 		}
 	}
 }
+/**
+ * Comparator to ensure correct ordering of percents
+ */
+class PercentComparator implements Comparator<String>, Serializable {
```
Comment (steveswinsburg) [inline gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/SettingsGradingSchemaPanel.java]: I've split all of these comparators out into their own classes. Could you do the same?

Comment (mpellicer) [inline gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/SettingsGradingSchemaPanel.java]: @steveswinsburg I still see the LetterGradeComparator inside the panel file, do you want me to move this to a separated file?

Thanks.

**Comment by mpellicer** [comment]
@steveswinsburg I've split the two comparators in separate files.

Regards.

**Comment by ottenhoff** [comment]
@mpellicer can you rebase from latest master?

**Comment by mpellicer** [comment]
@ottenhoff Done!

**Comment by mpellicer** [comment]
Rebased again, sorry for pushing a bit, can i do something else to help?

Thanks.

**Comment by ottenhoff** [comment]
Okay LGTM. @steveswinsburg  let's merge?

**Comment by mpellicer** [comment]
Thanks @ottenhoff, finally merged!

**Pull request merged**